routes/licenses.js

```diff
@@ -17,15 +22,15 @@ routes.push({
     notes: 'Returns a List of all Licenses',
     validate: {},
     response: {
-      schema: Joi.array().items(
-        Joi.object().keys({
-          code: Joi.string(),
-          url: Joi.string(),
-        })
-      ),
+      schema: Joi.array().items(licenseSchema),
     },
   },
-  handler: async (request, h) => h.response(mockResponse),
+  handler: async (request, h) => {
+    const licenseStore = request.server.app.services.licenses;
+    const licenses = licenseStore.all();
+    const response = licenses.map(({ url, name }) => ({ url: encodeURI(url), code: name }));
```

Comment:
To be compatible with the `Joi.string().uri()` validation the url may not contain whitespaces, so I went with encoding it here. We will have to clarify if that is the actual behaviour we want!